setup.py

```diff
@@ -1,34 +1,46 @@
-# -*- coding: utf-8 -*-
-"""
-    Setup file for kobuddy.
-    Use setup.cfg to configure your project.
+# see https://github.com/karlicoss/pymplate for up-to-date reference
 
-    This file was generated with PyScaffold 3.2.
-    PyScaffold helps you to put up the scaffold of your new Python project.
-    Learn more under: https://pyscaffold.org/
-"""
-import sys
 
-from pkg_resources import require, VersionConflict
-from setuptools import setup
+from setuptools import setup, find_packages # type: ignore
 
-try:
-    require('setuptools>=38.3')
-except VersionConflict:
-    print("Error: version of setuptools is too old (<38.3)!")
-    sys.exit(1)
 
-
-if __name__ == "__main__":
+def main():
+    pkgs = find_packages('src')
+    [pkg] = pkgs
```

Comment:
Can I ask what is is meant to do?

Reply:
hi! it's just so package name is inferred automatically, so I don't have to type it multiple times. Might be an overkill, but useful when maintaining a lot of small python packages

Reply:
Oh I see the list form is a way of unpacking!  It wasn’t as obvious to me as `(pkg,) = pkgs` or `pkg = pkgs[0]` so I wondered if it was a mistake.